Resources/doc/filter/range.md

```diff
@@ -1,67 +1,116 @@
+                "field": "wight"
+            }
+        }
+    }
+}
+
+```
+
+So lets say we execute request with `http://127.0.0.1?weight=13;56`. Notice that there
+are two variables separated by `;`. This indicates the range and it is the correct 
+format for the request. If the request will not contain `;` sign, the filter will become
+inactive. In this particular case though, filter state becomes active and executes 
+the following query:
+
+```json
+
+{
+    "post_filter": {
+        "range": {
+            "weight": {
+                "gte": 13,
+                "lte": 56
+            }
+        }
+    },
+    "aggregations": {
+        "weight": {
+            "stats": {
+                "field": "weight"
+            }
+        }
+    }
+}
 
-View data returned by this filter to be used in template:
+```
+
+Should be mentioned, that the `gte` and `lte` are used in stead of `gt` and `lt` 
+because the `inclusive` setting is set to `true` in configuration.
+
+> Take a look in the [basics topic](http://docs.ongr.io/FilterManagerBundle/Basics) how to pass `Request` object to the controller for filter execution.
+
+### Usage in the templates
+
+`Range` returns `RangeAwareViewData` as the result set for view data. Check the available functions in this class.
+
+`RangeAwareViewData` specific methods:
  
-| Method                  | Value                                            | 
-|-------------------------|--------------------------------------------------|
-| getName()               | Filter name                                      |
-| getResetUrlParameters() | Url parameters required to reset filter          |
-| getState()              | Filter state                                     |
-| getUrlParameters()      | Url parameters representing current filter state |
-| getMinBounds()          | Minimal value in all documents in provided field |
-| getMaxBounds()          | Maximal value in all documents in provided field |
-| getTags()               | Lists all tags specified at filter configuration |
-| hasTag($tag)            | Checks if filter has the specific tag            |
-
-Usage of this filter in the twig template:
+| Method                  | Value                                               | 
+|-------------------------|-----------------------------------------------------|
+| getMinBounds()          | Returns minimal value form the current document set |
+| getMaxBounds()          | Returns maximal value form the current document set |
 
-```yaml
+To use this filter in twig you have to provide a way to add the range to the request query.
+The methods from `RangeAwareViewData` help you to figure out the range that your current data
+set has, but the range itself needs to be specified by manually. The example below shows an
+example of how you can define a list of weight ranges.
+
+```twig
 
 <ul>
-    {% set filter = filter_manager.getFilters().wage %}
+    {% set filter = filter_manager.getFilters().weight %}
```

Comment:
Notice that in basics topic controller example already provides `filters` variable. We should use it in all docs continiously. 

btw according `Twig` there is a bad practice to call functions in the template. If you need somtehing just call the variable (if it's private twig will form getter).